apps/pie-microsite/src/content/pages/designers/how-we-support-you.md

```diff
@@ -0,0 +1,130 @@
+
+If you’re unsure about which model might be best for you or your team, or would like to know more about how we can support you, please [get in touch with us]().
+
+### Mentoring
+
+With the mentoring model, the business area designer is in charge of creating the component following the [contribution process]() outlined in our [Contribution]() section.
+
+During this process, a PIE designer will support the business area designer by reviewing the work, offering help and providing feedback at any stage.
+
+You can check the process closely in our [Process]() file.
+
+{% contentPageImage {
+src: "../../../../assets/img/designers/how-we-support-you/mentoring.svg",
+mobileSrc: "../../../../assets/img/designers/how-we-support-you/mentoring_narrow.svg",
+width: "800px",
+caption: "A map of how Libraries are structured in Figma."
+} %}
+
+### Cleanup
+
+With the cleanup model, the business area designer creates the component only **requesting support if/when needed**. Designers must ensure they create the new component following the [contribution process]() outlined in our [Contribution]() section.
+
+Once the component is finished, a PIE designer will help the business area designer finalise and finesse the component, making sure it aligns with our design system.
+
+You can check the process closely in our [Process]() file.
+
+{% contentPageImage {
+src: "../../../../assets/img/designers/how-we-support-you/cleanup.svg",
+mobileSrc: "../../../../assets/img/designers/how-we-support-you/cleanup_narrow.svg",
+width: "800px",
+caption: "A flow diagram which shows the steps we follow during our Cleanup support model."
+} %}
+
+### Partnership
+
+With the partnership model, the business area designer **will work closely with a PIE designer** throughout the component creation process, from research and testing, right through component building and finalisation.
+
+You can check the process closely in our [Process]() file.
+
+{% contentPageImage {
+src: "../../../../assets/img/designers/how-we-support-you/partnership.svg",
+mobileSrc: "../../../../assets/img/designers/how-we-support-you/partnership_narrow.svg",
+width: "800px",
+caption: "A flow diagram which shows the steps we follow during our Partnership support model."
+} %}
+
+---
+
+## The role of the PIE designer
+
+PIE designers are specialists in systems thinking and design systems maintenance. Our role is not just to define foundations, components, patterns and documentation, but also provide you with guidance on how to use PIE and how to implement it consistently throughout your products.
+
+As a result of that, we’re often asked how PIE designers are usually involved when working together with business area designers. To help describe it, we’ve put together a list of the things we do and don’t as part of our role:
+
+### What we do:
+
+{% list {
+    type: listTypes.icon,
+    items: [
+        {
+          iconName: 'CheckCircleFilledSmall',
```

Comment:
do we anticipate needing a different type of icon per list item? if not we could reduce some duplication here and just specify what icon to use for the entire list